bluebinder.c

```diff
@@ -514,7 +529,8 @@ int main(int argc, char *argv[])
     death_id = gbinder_remote_object_add_death_handler
             (remote, binder_remote_died, &proxy);
 
-    g_main_loop_run(proxy.loop);
+    if(!proxy.init_failed)
```

Comment:
nitpick: space missing between "if" and "("

Reply:
Hi, thanks a lot, I've fixed that